smartsim/ml/tf/utils.py

```diff
@@ -41,7 +41,7 @@ def freeze_model(
 
     to use a Keras or TensorFlow model in SmartSim, the model
     must be frozen and the inputs and outputs provided to the
-    smartredis.client.set_model_from_file() method.
+    None.client.set_model_from_file() method.
```

Comment:
This whole paragraph seems very SSv0.7-centric. I would just remove it entirely!